### PR TITLE
dev: reduce processed fields clutter in list view

### PR DESCRIPTION
**Pull request by carmichaelong**
shorten some column names and clean up duration formatting

### CODE REVIEW

File: mcserver/admin.py [hunk 1]
```diff
@@ -102,16 +103,23 @@ class TrialAdmin(admin.ModelAdmin):
...
         'created_at', 'updated_at',
         'server', 'git_commit',
-        'processed_duration', 'processed_count',
+        'formatted_duration', 'processed_count',
         'is_meta_null',
         'trashed', 'trashed_at',
     )
     raw_id_fields = ('session',)
     inlines = [ResultInline]
+    formatted_duration.short_description = 'duration'
```
Comment (AlbertoCasasOrtiz) [inline mcserver/admin.py:112]: I tried to run it locally and it is complaining about `formated_duration `and `processed_duration` not being defined:

```
  File "C:\Workspaces\opencap\opencap-api\mcserver\admin.py", line 97, in <module>
    class TrialAdmin(admin.ModelAdmin):
  File "C:\Workspaces\opencap\opencap-api\mcserver\admin.py", line 112, in TrialAdmin
    formatted_duration.short_description = 'duration'
NameError: name 'formatted_duration' is not defined
```

**Review by AlbertoCasasOrtiz — CHANGES_REQUESTED** [review]
I added a comment with the error the API is throwing locally.

**Comment by carmichaelong** [comment]
@AlbertoCasasOrtiz thanks for the catch (i should set up a local build at some point). pushed a commit that should fix the issue

**Comment by AlbertoCasasOrtiz** [comment]
It fixed it! I got then the same problem with processed_count and added a function too so it recognizes it.

Now, when I go to admin console, I got this:

```
   File "C:\Workspaces\opencap\opencap-api\mcserver\admin.py", line 117, in formatted_duration
     hours, remainder = divmod(int(obj.duration.total_seconds()), 3600)
 AttributeError: 'Trial' object has no attribute 'duration'
 [02/Oct/2024 11:13:48] "GET /admin/mcserver/trial/ HTTP/1.1" 500 452129
```

Looks like the `obj.duration` inside of the formatted_duration function is not being recognized.

**Comment by carmichaelong** [comment]
Thanks for checking that. Makes sense that it didn't work. I pushed a change because the field name is `processed_duration` (rather than just `duration`).

**Comment by AlbertoCasasOrtiz** [comment]
It was failing now, but just found what the problem was. Since processed duration can be null or blank, it could not access to the total_seconds() for a non-defined object.

I just added an if-else (I returned the object as is without formatting in the case it is undefined, so check if that is what we want).

**Comment by carmichaelong** [comment]
Awesome, yes, that's the formatting that I was going for! LGTM.

**Review by AlbertoCasasOrtiz — APPROVED** [review]
Everything LGTM after the changes, approving!

**Pull request merged**